internal/command/apps/create.go

```diff
@@ -80,6 +82,8 @@ func RunCreate(ctx context.Context) (err error) {
 			aName, fName)
 
 		return
+	case ctxName != "":
+		name = ctxName
```

Comment:
I like it but this is a different thing right? it creates the name passed by `-a` flag or found in local fly.toml, is it?

Reply:
should it check that the ctxName doesn't differ from the other names `aName` and `fName` ?

the context name can come from FLY_APP envvar, `-a` and local fly.toml. 

it can make `fly apps create myappname` fail or pick the wrong name if run in any of those conditions.

Reply:
oh interesting! I hadn't thought of that, I'll add that to the above conditional